custom_components/powercalc/data/osram/Classic A60 RGBW/model.json

```
@@ -0,0 +1,13 @@
+{
+    "calculation_strategy": "lut",
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "Shelly Plus Plug S",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 2,
+        "SLEEP_TIME": 3,
+        "VERSION": "v1.9.2:docker"
+    },
+    "name": "Extended color light (Classic A60 RGBW)",
```

Is it this light? https://www.ledeshop.net/osram-lightify-classic-a60-rgbw-4052899926097-led-bulb.html
Does it also state lightify on the bulb.
In that case I think following name would be nice.

```suggestion
    "name": "Lightify Classic A60 RGBW",
```

---

Yes, this seems to be the right bulb